kubos/utils/git_utils.py

```diff
@@ -0,0 +1,138 @@
+# Kubos SDK
+# Copyright (C) 2016 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import git
+import logging
+import packaging.version
+import os
+import sys
+
+from kubos.utils import sdk_utils
+from kubos.utils.constants import *
+
+def get_repo(path):
```

Comment:
Does this function need to return both repo and origin? It looks like the callers only ever use repo.